framework/Models/ROM.py

```diff
@@ -174,8 +174,8 @@ class cls. This one seems a bit excessive, are all of these for this class?
     inputSpecification.addSub(InputData.parameterInputFactory("random_start", InputData.IntegerType))
     # GaussPolynomialROM and HDMRRom
     inputSpecification.addSub(InputData.parameterInputFactory("IndexPoints", InputData.StringType))
-    inputSpecification.addSub(InputData.parameterInputFactory("IndexSet",IndexSetInputType))
-    inputSpecification.addSub(InputData.parameterInputFactory('pivotParameter',contentType=InputData.StringType))
+    inputSpecification.addSub(InputData.parameterInputFactory("IndexSet", IndexSetInputType))
```

Comment:
you should use ```contentType```, otherwise the type will not be used.

Reply:
wait, so all of these in the ROM are wrong?? This should be noted to fix generally somewhere. I added it here.

Reply:
I think I have fix this in one of my branch (for the deep neuron network), regarding this, I just want you to update the modifications that you made.